app/phpunit.xml.dist

```diff
@@ -29,7 +30,9 @@
 
         <!-- Complete suite for PIM integration tests -->
         <testsuite name="PIM_Integration_Test">
-            <file></file>
+            <directory suffix="Integration.php">../src</directory>
```

Comment:
this phpunit file seems useless

Reply:
I have reused the same file than from enterprise standard.
But indeed, it doesn't really make sense as phpunit is not even in the composer.json file of the standard ;)